app/services/spree/netsuite_update_line_item_service.rb

```diff
@@ -52,7 +52,7 @@ def update(order)
           items: items
         },
         orderstatus: payment[:status],
-        shipmethod: { id: 66857 },
+        shipmethod: { id: 66859 },
```

Comment:
need to dynamic this value expose new field on Netsuite settings on admin pannel. "shipping_method_id"